build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.3'
+        classpath 'com.android.tools.build:gradle:3.0.0'
```

Comment:
is 3.0.0 already quite stable? 
better to change to 3.1.x if it exists or leave 2.3.3